src/components/common/OfficeAddresses.vue

```diff
@@ -243,34 +243,35 @@ export default class OfficeAddresses extends Mixins(CommonMixin) {
   }
 
   // The 4 addresses that are the current state of the BaseAddress components:
-  private mailingAddress = {} as AddressIF
-  private deliveryAddress = {} as AddressIF
-  private recMailingAddress = {} as AddressIF
-  private recDeliveryAddress = {} as AddressIF
+  protected mailingAddress = {} as AddressIF
+  protected deliveryAddress = {} as AddressIF
+  protected recMailingAddress = {} as AddressIF
+  protected recDeliveryAddress = {} as AddressIF
 
   // Validation events from BaseAddress:
-  private mailingAddressValid: boolean = true
-  private deliveryAddressValid: boolean = true
-  private recMailingAddressValid: boolean = true
-  private recDeliveryAddressValid: boolean = true
+  private mailingAddressValid = true
+  private deliveryAddressValid = true
+  private recMailingAddressValid = true
+  private recDeliveryAddressValid = true
 
-  private inheritMailingAddress: boolean = true
+  /** State of the checkbox for determining whether the Delivery address is the same as the Mailing address. */
+  protected inheritMailingAddress = true
 
-  // State of the checkbox for determining whether or not the mailing address is the same as the delivery address
-  // For Records Office
-  private inheritRecMailingAddress: boolean = true
+  /**
+   * State of the checkbox for determining whether or not the mailing address is the same as the delivery address.
+   * For Records Office.
+   */
+  private inheritRecMailingAddress = true
 
-  // State of the checkbox for determining whether the Record address is the same as the Registered address
-  private inheritRegisteredAddress: boolean = true
+  /** State of the checkbox for determining whether the Record address is the same as the Registered address. */
+  private inheritRegisteredAddress = true
```

Comment:
The comment in JS Doc format allows VS Code to display the comment when hovering over the variable.